runtime/src/xcm_config.rs

```diff
@@ -138,16 +134,25 @@ pub type XcmRouter = WithUniqueTopic<(
     ChildParachainRouter<Runtime, XcmPallet, PriceForChildParachainDelivery>,
 )>;
 
+// zkVerify-EVM-Parachain
+pub const ZKV_EVM_PARA_ID: u32 = 1;
+// paratest, included in this repository
 pub const TEST_PARA_ID: u32 = 1599;
+
 parameter_types! {
-    pub const Acme: AssetFilter = Wild(AllOf { fun: WildFungible, id: AssetId(TokenLocation::get()) });
+    pub const TVFYAsset: AssetFilter = Wild(AllOf { fun: WildFungible, id: AssetId(TokenLocation::get()) });
     pub TestParaLocation: Location = Parachain(TEST_PARA_ID).into_location();
```

Comment:
Is there a way around the fact that we are keeping here stuff related to the Para testing ?

Reply:
Not sure, the paratest must be whitelisted from the relay point of view, so the info must be available somewhere in the relay chain code.

Maybe we can change our Paratest id to `1` as well, and pretend that Paratest is the same as the evm para? But honestly I don't like that route